src/edit/Toolbar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-toolbar
+    height="72"
+    color="transparent"
+    class="elevation-0">
+    <v-toolbar-title class="pl-1">Api Video</v-toolbar-title>
+    <v-toolbar-items class="mx-auto">
+      <upload-btn
+        v-if="!fileName"
+        @change="upload"
+        label="Upload Api video"
+        accept="video/mp4"
+        class="upload-btn" />
+      <template v-else>
+        <v-text-field
+          :value="fileName"
+          readonly hide-details filled />
+      </template>
+    </v-toolbar-items>
+  </v-toolbar>
+</template>
+
+<script>
+import get from 'lodash/get';
+import UploadBtn from './UploadBtn.vue';
+
+const MP4_MIME_TYPE = 'video/mp4';
+const FORMAT_ERROR = 'MP4 format is required.';
+
+export default {
+  name: 'tce-api-video-toolbar',
+  inject: ['$elementBus'],
+  props: {
+    element: { type: Object, required: true }
+  },
+  computed: {
+    fileName: ({ element }) => get(element, 'data.fileName', '')
+  },
+  methods: {
+    upload(e) {
+      const file = e.target.files[0];
+      if (file.type !== MP4_MIME_TYPE) {
+        this.$elementBus.emit('error', { error: { message: FORMAT_ERROR } });
+        return;
+      }
+      this.$elementBus.emit('save', { file });
+    }
+  },
+  components: { UploadBtn }
+};
+</script>
+
+<style lang="scss" scoped>
+.v-toolbar__title {
+  min-width: 23.875rem;
+  text-align: left;
+}
+
+.upload-btn ::v-deep .v-btn {
+  height: 100%;
+
+  .v-btn__content {
```

Comment:
Think this might work:
```suggestion
  &__content {
```